### PR TITLE
[BUGZ-352] :: Audio settings persisting

### PR DESCRIPTION
**Pull request by MiladNazeri**
https://highfidelity.atlassian.net/browse/BUGZ-352

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20698/HighFidelity-Beta-PR16013-cf7cff8.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20698/HighFidelity-Beta-Interface-PR16013-cf7cff8.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20698/HighFidelity-Beta-PR16013-cf7cff8.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20698/HighFidelity-Quest-Beta-PR16013-cf7cff8.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20698/HighFidelity-Beta-PR16013-cf7cff8.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20698/HighFidelity-Beta-Interface-PR16013-cf7cff8.exe

File: scripts/simplifiedUIBootstrapper.js [hunk 1]
```diff
@@ -62,6 +62,23 @@ function runDefaultsTogether() {
 }
 
 
+Audio.receivedFirstPacket.connect(onReceivedFirstPacket);
+function onReceivedFirstPacket() {
```
Comment (ZappoMan) [inline scripts/simplifiedUIBootstrapper.js]: this feels wrong... I mean, what if you never connect to a domain, then I think that means we wouldn't ever set the values?

**Comment by MiladNazeri** [comment]
@ZappoMan New approach.  This one works for both versions.

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20707/HighFidelity-Beta-PR16013-e1428e6.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20707/HighFidelity-Beta-Interface-PR16013-e1428e6.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20707/HighFidelity-Beta-PR16013-e1428e6.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20707/HighFidelity-Quest-Beta-PR16013-e1428e6.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20707/HighFidelity-Beta-PR16013-e1428e6.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20707/HighFidelity-Beta-Interface-PR16013-e1428e6.dmg

**Comment by salinacsoto** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20721/HighFidelity-Beta-PR16013-099eda0.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20721/HighFidelity-Quest-Beta-PR16013-099eda0.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20721/HighFidelity-Beta-PR16013-099eda0.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20721/HighFidelity-Beta-Interface-PR16013-099eda0.dmg

**Comment by salinacsoto** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20725/HighFidelity-Beta-PR16013-099eda0.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20725/HighFidelity-Beta-Interface-PR16013-099eda0.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20725/HighFidelity-Beta-PR16013-099eda0.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20725/HighFidelity-Quest-Beta-PR16013-099eda0.apk)

**Comment by MiladNazeri** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20729/HighFidelity-Beta-PR16013-099eda0.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20729/HighFidelity-Beta-Interface-PR16013-099eda0.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20729/HighFidelity-Beta-PR16013-099eda0.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20729/HighFidelity-Quest-Beta-PR16013-099eda0.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20729/HighFidelity-Beta-PR16013-099eda0.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20729/HighFidelity-Beta-Interface-PR16013-099eda0.exe

**Comment by MiladNazeri** [comment]
closing this pr in favor of https://github.com/highfidelity/hifi/pull/16021

**Pull request closed without merging**